test/interactions/interactionTests.ts

```diff
@@ -0,0 +1,110 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Interactions", () => {
+  describe("Interaction", () => {
```

Comment:
Okay, that makes a lot of sense. It might be good to come up with some way to identify the base class in tests, but I think we can save that decision for when we revisit tests.